WordPress/gutenberg
# Tweak border control button to proper metrics and simpler action

## What?
Minor fix to adjust the reset and style buttons to fit within the updated metrics (40px and 32px) buttons—as detailed in https://github.com/WordPress/gutenberg/issues/46734. Also uses the correct border-top color now.

## Testing Instructions

1. Open a post or page.
2. Insert a group block.
3. Add a border. 
4. See the style buttons now 32px, instead of 30px.
5. See the 46px "Reset to default" button now "Reset" and 40px.

## Screenshots or screencast 

| Before  | After |
| ------------- | ------------- |
|<img width="262" alt="CleanShot 2023-08-28 at 12 43 53" src="https://github.com/WordPress/gutenberg/assets/1813435/2a52b1a7-fcaf-4b25-9847-406392bea925">|<img width="262" alt="CleanShot 2023-08-28 at 12 53 28" src="https://github.com/WordPress/gutenberg/assets/1813435/acc8699f-a2e9-4fc1-a1b2-43b3a1048c52">|
